packages/design/src/radii.css

```diff
@@ -1,7 +1,7 @@
 :root {
-  --radius-small: var(--space-minuscule);
-  --radius-base: var(--space-smallest);
-  --radius-large: var(--space-smaller);
-  --radius-larger: var(--space-small);
+  --radius-small: calc(var(--radius-base) / 2);
```

Comment:
Rationale for moving these away from space tokens:

1. Radius ≠ space
2. Radius is its' own sub-system that is largely aesthetic. Basing the other values off of `base` allows for simple re-working of the radius system within a consuming scenario.

Reply:
Love this!